uiautomationutilities/perf_tool/readme.md

```diff
@@ -8,148 +8,106 @@
 
```

Comment:
There is some terminology used in this document which I think could use a little definition for newcomers/maintainers. Specifically, concepts like `baseFiles` and `targetFiles` are mentioned. As I understand it, a `baseFile` is a "first run" against which subsequent runs (targets?) are compared... can we define these terms explicitly within this document?

Reply:
Done